iothub/device/src/Transport/Mqtt/MqttTransportSettings.cs

```diff
@@ -157,8 +160,15 @@ public bool CertificateRevocationCheck
 
         /// <summary>
         /// The time to wait for receiving an acknowledgment for a CONNECT packet.
-        /// The default is 10 seconds.
+        /// The default is 60 seconds.
         /// </summary>
+        /// <remarks>
+        /// In the event that IoT Hub receives burst traffic, it will implement traffic shaping in order to process the incoming requests.
+        /// In such cases, during client connection the CONNECT requests can have a delay in being acknowledged and processed by IoT Hub.
+        /// The <see cref="ConnectArrivalTimeout"/> governs the duration the client will wait for a CONNACK packet before disconnecting and reopening the connection.
```

Comment:
Self-referenced type in web docs just links back to current page. I think that is a bad experience for users, so no point in adding the link.

Reply:
Maybe just setting it as "code" will convey what you want.

```suggestion
        /// The <c>ConnectArrivalTimeout</c> governs the duration the client will wait for a CONNACK packet before disconnecting and reopening the connection.
```

Reply:
It will. I wanted to see the "commit suggestions" in action, but I'll have to leave that for another day.